app/models/grda_warehouse/hud/project.rb

```diff
@@ -322,6 +324,10 @@ def serves_children?
       # END_ACL
     end
 
+    def can?(user, permission: :can_view_projects)
```

Comment:
@dtgreiner this is what I was thinking of in relation to the homeless verification and history changes you were working on.